### ISSUE TITLE
eJabberd admin WebUI blocked by default Hub/AWS firewall/security rules

### ISSUE THREAD

**Issue by JedMeister** [turnkeylinux/tracker#283]
As the title says the eJabberd appliance admin WebUI (http://<ip-or-fqdn>:5280/admin) is blocked by default firewall/security rules. Adding it - as stated in (this post)[http://www.turnkeylinux.org/forum/support/20141204/ejabberd-install-issue#comment-21955] - resolves that.

**Comment by JedMeister** [comment]
ejabberd appliance currently deferred from v14.0 release so retargeting to "on ice"

**Comment by JedMeister** [comment]
ejabberd appliance has been discontinued
